### PR TITLE
Support for typescript4.4, needs catch(error) to be specified of type Error:any to be printed (defaults to type unknown)

### PR DESCRIPTION
**Pull request by spencerbug**
## Description
https://github.com/dfinity/internet-identity/issues/417

## Motivation and Context
I needed this change to compile internet-identity

## How Has This Been Tested?
This has not gone through any standard regression testing for internet identity
but I have been running this locally with no issues so far. 

## Types of changes
compilation fix

## Checklist:
 
- [x] My code follows the code style of this project.
- [ ]  My change requires a change to the documentation.
- [ ]  I have updated the documentation accordingly.
- [ ]  I have added tests to cover my changes.
- [ ]  All new and existing tests passed.

### CODE REVIEW

File: src/frontend/src/flows/login.ts [hunk 1]
```diff
@@ -61,12 +61,12 @@ export const login = async (
...
       }
     }
-  } catch (err) {
+  } catch (err:any) {
     await displayError({
       title: "Something went wrong",
       message:
-        "An unexpected error occurred during authentication. Please try again",
-      detail: err,
+        "An unexpected error occured during authentication. Please try again",
```
Comment (nmattia) [inline src/frontend/src/flows/login.ts:68]: Mh, I don't think that's correct: 

> Spelling words isn’t always easy, especially when they contain double letters. In this case, the correct spelling is occurred. How can you remember it when you need it?
> 
> The best way to remember how to spell occurred is to remember its double set of double consonants. In English, the final letter is doubled when a word of two or more syllables has stress on the final syllable. Occur fits the rule, so there are two Cs and two Rs in occurred.

https://www.grammarly.com/blog/occurred-occured-ocurred/

**Review by nmattia — COMMENTED** [review]
One comment; thanks for the PR!

File: src/frontend/src/flows/login.ts (hunk 1 above)
Comment (spencerbug) [reply src/frontend/src/flows/login.ts]: Woops, I think that happened from a merge conflict. Updated

**Comment by nmattia** [comment]
@spencerbug looks like CI tried to fix the formatting for you but failed to push because you've created a PR from your fork; would you mind running `npm run format` locally? Thanks!

**Comment by spencerbug** [comment]
> 

@nmattia I updated the PR with the linter. Sorry for the delay

**Comment by nmattia** [comment]
Hi @spencerbug , no worries, thanks for your contribution!

**Review by nmattia — APPROVED** [review]
LGTM!

**Pull request merged**